src/obs-ndi-source.cpp

```diff
@@ -440,6 +440,16 @@ void *ndi_source_thread(void *data)
 
 	NDIlib_recv_create_v3_t *reset_recv_desc = &recv_desc;
 
+	if (!s->running) {
```

Comment:
I don't get this.
This line is:
```
if (!s->running) { ... }
```
The next immediate line after this if statement is:
```
while (s->running) { ... }
```

If the `if` statement is entered, then the while loop never will be entered.
If the `if` statement isn't entered, then the while loop will be entered. 

I know this `ndi_source_thread` runs inside of the while loop.

So, this tells me that this `if (!s->running) { ... }` statement is not doing anything.

😕